src/gsad_gmp.c

```diff
@@ -5497,19 +5561,21 @@ create_target_gmp (gvm_connection_t *connection, credentials_t *credentials,
 
   command = g_strdup_printf (
     "<create_target>"
-    "%s%s%s%s%s%s%s%s"
+    "%s%s%s%s%s%s%s%s%s"
     "</create_target>",
     xml->str, comment_element, ssh_credentials_element,
     ssh_elevate_credentials_element ? ssh_elevate_credentials_element : "",
     smb_credentials_element, esxi_credentials_element, snmp_credentials_element,
-    asset_hosts_element);
+    krb5_credentials_element, asset_hosts_element);
```

Comment:
This may be NULL. Alternatively you can set it to an empty string above instead of NULL
```suggestion
    krb5_credentials_element ? krb5_credentials_element : "", asset_hosts_element);
```